node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -3249,6 +3557,26 @@ mod tests {
     #[test]
     fn initial_standard_gossip_does_not_produce_neighborship_if_destination_degree_is_already_full()
     {
+        /*
+        Destination Database ==>
+            N2  N3
+             \  /
+        N1---Dest---Third
+             /  \
+            N4  N5
+
+        Source Database ==>
+        Src---Third---Dest
+
+        Disc
+
+        Expected Destination Database ==>
```

Comment:
This diagram may be incorrect.

Reply:
I made you left this comments. After checking one more time, I realised that the diagram is accurately depicting what we're doing in the test.